ext/date/php_date.c

```diff
@@ -2363,6 +2363,11 @@ static HashTable *date_object_get_properties_timezone(zval *object) /* {{{ */
 	return props;
 } /* }}} */
 
+static inline void date_object_update_properties_timezone(zval *object) /* {{{ */
```

Comment:
Yep. IMHO I'd like to rewrite these parts of code, since I think get_properties should do retrieve only, updating data inside should have been forbidden. but it's really a huge project, and modifying a code which still can work is not a good idea, so I just leave them alone.